### PR TITLE
System.IO.sln hangs

### PR DESCRIPTION
**Pull request by carlossanlop**
When opening the System.IO.sln file with Visual Studio, it would hang attempting to load. Some of the Project GUIDs were incorrectly placed.

cc @scalablecory - thanks for noticing this.

### CODE REVIEW

File: src/libraries/System.IO/System.IO.sln [hunk 1]
```diff
@@ -2,17 +2,17 @@ Microsoft Visual Studio Solution File, Format Version 12.00
...
 MinimumVisualStudioVersion = 10.0.40219.1
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.IO.Tests", "tests\System.IO.Tests.csproj", "{492EC54D-D2C4-4B3F-AC1F-646B3F7EBB02}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.IO.Tests", "tests\System.IO.Tests.csproj", "{BC2E1649-291D-412E-9529-EDDA94FA7AD6}"
 	ProjectSection(ProjectDependencies) = postProject
-		{07390899-C8F6-4e83-A3A9-6867B8CB46A0} = {07390899-C8F6-4e83-A3A9-6867B8CB46A0}
+		{5471BFE8-8071-466F-838E-5ADAA779E742} = {5471BFE8-8071-466F-838E-5ADAA779E742}
 	EndProjectSection
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.IO", "src\System.IO.csproj", "{07390899-C8F6-4e83-A3A9-6867B8CB46A0}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.IO.", "src\System.IO.csproj", "{5471BFE8-8071-466F-838E-5ADAA779E742}"
```
Comment (jkotas) [inline src/libraries/System.IO/System.IO.sln]: Extra `.` after `IO` ?

File: src/libraries/System.IO/System.IO.sln [hunk 2]
```diff
@@ -2,17 +2,17 @@ Microsoft Visual Studio Solution File, Format Version 12.00
...
 	ProjectSection(ProjectDependencies) = postProject
-		{88883C57-83BE-4E93-A363-4CFC716F248F} = {88883C57-83BE-4E93-A363-4CFC716F248F}
+		{4ADD9456-A929-4254-B8A2-16FC628ABFDA} = {4ADD9456-A929-4254-B8A2-16FC628ABFDA}
 	EndProjectSection
 EndProject
-Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.IO", "ref\System.IO.csproj", "{88883C57-83BE-4E93-A363-4CFC716F248F}"
+Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.IO.", "ref\System.IO.csproj", "{4ADD9456-A929-4254-B8A2-16FC628ABFDA}"
```
Comment (jkotas) [inline src/libraries/System.IO/System.IO.sln]: Dtto

**Comment by danmosemsft** [comment]
Worth a bug with callstack at https://github.com/dotnet/project-system/issues for the hang?

File: src/libraries/System.IO/System.IO.sln (hunk 1 above)
Comment (carlossanlop) [reply src/libraries/System.IO/System.IO.sln]:
```suggestion
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.IO", "src\System.IO.csproj", "{5471BFE8-8071-466F-838E-5ADAA779E742}"
```

File: src/libraries/System.IO/System.IO.sln (hunk 2 above)
Comment (carlossanlop) [reply src/libraries/System.IO/System.IO.sln]:
```suggestion
Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "System.IO", "ref\System.IO.csproj", "{4ADD9456-A929-4254-B8A2-16FC628ABFDA}"
```

**Review by Anipik — APPROVED** [review]

**Pull request merged**